### DISCUSSION TITLE
怎么从【@umijs/max】导出【RouteObject】等类型

### DISCUSSION THREAD

**Discussion by seafronthu** [umijs/umi#10505] [Q&A] [answered]
我使用的是@umijs/max，我想导出关于路由相关的类型，该如何导出

**Comment by fz6m** [comment, marked as answer]
这个类型只是用来类型计算的中间变量，没有应用意义，不提供导出。

如果你需要配置文件里的 `routes` 类型，可以把这部分提取出来，在配置文件和项目中同时导入，然后用 `typeof routes` 获取。
